diff -urN cyclone-0.5-0.5.orig/Makefile cyclone-0.5-0.5/Makefile
--- cyclone-0.5-0.5.orig/Makefile	2003-04-11 22:48:36.000000000 +0200
+++ cyclone-0.5-0.5/Makefile	2003-08-11 15:38:15.000000000 +0200
@@ -341,7 +341,7 @@
 
 ifdef INC_INSTALL
 inc_install:
-	$(SHELL) config/cyc_install include/* $(INC_INSTALL)
+	$(SHELL) config/cyc_install include/* $(DESTDIR)$(INC_INSTALL)
 inc_uninstall:
 	$(SHELL) config/cyc_install -u $(INC_INSTALL)
 else
@@ -350,7 +350,7 @@
 endif
 ifdef BIN_INSTALL
 bin_install:
-	$(SHELL) config/cyc_install bin/cyclone$(EXE) bin/cycbison$(EXE) bin/cyclex$(EXE) bin/cycflex$(EXE) bin/rewrite$(EXE) bin/aprof$(EXE) $(BIN_INSTALL)
+	$(SHELL) config/cyc_install bin/cyclone$(EXE) bin/cycbison$(EXE) bin/cyclex$(EXE) bin/cycflex$(EXE) bin/rewrite$(EXE) bin/aprof$(EXE) $(DESTDIR)$(BIN_INSTALL)
 bin_uninstall:
 	$(SHELL) config/cyc_install -u $(BIN_INSTALL)
 else
@@ -359,7 +359,7 @@
 endif
 ifdef LIB_INSTALL
 lib_install:
-	$(SHELL) config/cyc_install bin/lib/* $(LIB_INSTALL)
+	$(SHELL) config/cyc_install bin/lib/* $(DESTDIR)$(LIB_INSTALL)
 lib_uninstall:
 	$(SHELL) config/cyc_install -u $(LIB_INSTALL)
 else
