diff -ur cyclone/Makefile cyclone-/Makefile
--- cyclone/Makefile	Tue Nov 20 16:00:22 2001
+++ cyclone-/Makefile	Wed Dec 12 01:27:29 2001
@@ -65,7 +65,7 @@
 
 ifdef INC_INSTALL
 inc_install:
-	$(SHELL) config/cyc_install include/* $(INC_INSTALL)
+	$(SHELL) config/cyc_install include/* $(DESTDIR)$(INC_INSTALL)
 inc_uninstall:
 	$(SHELL) config/cyc_install -u $(INC_INSTALL)
 else
@@ -77,7 +77,7 @@
 
 ifdef BIN_INSTALL
 bin_install:
-	$(SHELL) config/cyc_install bin/cyclone$(EXE) bin/cycbison$(EXE) bin/cyclex$(EXE) bin/cycflex$(EXE) $(BIN_INSTALL)
+	$(SHELL) config/cyc_install bin/cyclone$(EXE) bin/cycbison$(EXE) bin/cyclex$(EXE) bin/cycflex$(EXE) $(DESTDIR)$(BIN_INSTALL)
 bin_uninstall:
 	$(SHELL) config/cyc_install -u $(BIN_INSTALL)
 else
@@ -89,7 +89,7 @@
 
 ifdef LIB_INSTALL
 lib_install:
-	$(SHELL) config/cyc_install bin/cyc-lib/* $(LIB_INSTALL)
+	$(SHELL) config/cyc_install bin/cyc-lib/* $(DESTDIR)$(LIB_INSTALL)
 lib_uninstall:
 	$(SHELL) config/cyc_install -u $(LIB_INSTALL)
 else
